doc/README.md

```diff
@@ -1,30 +1,18 @@
 [TerriaJS](http://terria.io) is an open-source framework for web-based geospatial catalog explorers.
 
-The documentation on this site is for applications using TerriaJS version 8. If you are still maintaining an application using TerriaJS version 7 or lower go to [docs-v7.terria.io](https://docs-v7.terria.io).
+-   [Getting Started](getting-started.md): Quick start guide to building your first TerriaJS application.
+-   [Customizing](customizing/README.md): Configure and tweak a TerriaJS application, including skinning and setting up the catalog.
+-   [Connecting to Data](connecting-to-data/README.md): Connect TerriaJS to your servers and data.
+-   [Deploying](deploying/README.md): Deploy a TerriaJS application in simple and advanced scenarios.
+-   [Contributing](contributing/README.md): Add new features to TerriaJS, be part of the TerriaJS development team, set up a development environment, write tests, and perform code reviews.
```

Comment:
Prettier reformats lists with extra spaces after bullets when using indent_size=4. It's a little ugly, but indent_size=4 is needed for mkdocs, and it's still better to use Prettier than to have to enforce indent_size=4 manually.